addons/project_purchase/tests/test_project_profitability.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime
+
+from odoo import Command
+from odoo.tests import tagged
+
+from odoo.addons.project.tests.test_project_profitability import TestProjectProfitabilityCommon
+from odoo.addons.purchase.tests.test_purchase_invoice import TestPurchaseToInvoice
+
+
+@tagged('-at_install', 'post_install')
+class TestProjectPurchaseProfitability(TestProjectProfitabilityCommon, TestPurchaseToInvoice):
```

Comment:
Pay attention it will relaunch the tests in `TestPurchaseToInvoice`. 😬

Reply:
I've extracted the setUpClass into it's own class to avoid triggering the other tests.
I don't know if this is how it is supposed to be done, but it gets the job done.